railgun/manager/run.go

```diff
@@ -110,6 +110,10 @@ func Run(ctx context.Context, c *config.Config) error {
 	}
 
 	// determine the proxy synchronization strategy
+	if c.ProxySyncSeconds < 3.0 {
```

Comment:
Hard code would be hard to debug. A const as following
const ProxySyncThreshold 3 
would be a better programming strategy.

Reply:
Good call, there's actually a variable already present for this purpose, I've updated this in:

3767ea8cd977002a1f592aed7f107bb79728ca31

:+1: